lib/bemxjst/index.js

```diff
@@ -320,6 +319,7 @@ BEMXJST.prototype.runUnescaped = function runUnescaped(context) {
 BEMXJST.prototype.runSimple = function runSimple(simple) {
   this.context._listLength--;
   var res = '';
+  /* istanbul ignore else */
```

Comment:
if `simple` quals `NaN` then output be empty string. It’s OK. And it’s behaviour was not changed.

I just inctructed Istanbul ignore `else` branch for `if` from 323 line.